src/AutoRest.CSharp/Mgmt/Decorator/SchemaExtensions.cs

```diff
@@ -24,6 +24,22 @@ private static IEnumerable<Property> GetAllProperties(this ObjectSchema schema)
             return schema.Parents!.All.OfType<ObjectSchema>().SelectMany(parentSchema => parentSchema.Properties).Concat(schema.Properties);
         }
 
+        public static bool HasTags(this Schema schema)
```

Comment:
Can we use this method in below method `IsTagsOnly`?

Reply:
These two are different. One of them is identifying if one of its properties is tags, the other one is if its properties contains tags.
I could introduce a helper method to identify if a property is `tags`. This will make some of the code in these two methods shared